app/views/include/forms2/textarea.scala.html

```diff
@@ -57,30 +57,34 @@ <h1>
         </div>
         <p>@labelSpan</p>
     } else {
-        <label class="form-label" for="@field.id">
+        <label class="govuk-form-label" for="with-hint">
             @labelSpan
         </label>
     }
 }
 
-<div class="form-group@if(charCount){ char-counter}@if(field.hasErrors) { form-field--error}" @if(charCount) {data-char-counter} @if(data){ data-reveal="@data"}>
-    @label
-    @if(hintText.nonEmpty) {
-        <span class="form-hint" id="@{field.id}-hint">@Messages(hintText)</span>
-    }
-    @if(field.hasErrors) {
-        <span class="error-notification" id="@{field.id}-error-notification" data-journey="error - field:user input:@field.name">
-            <span class="visuallyhidden">Error: </span>@field.error
-        </span>
-    }
-    <textarea @if(charCount){data-char-field} name="@field.name" id="@field.id"
-        @if(hintText.nonEmpty || field.hasErrors) { aria-describedby="@if(hintText.nonEmpty) {@{field.id}-hint} @if(field.hasErrors) {@{field.id}-error-notification}" }
-        @if(cols.nonEmpty) { cols="@cols" }
-        @if(rows.nonEmpty) { rows="@rows" }
-        @if(maxLength.nonEmpty) { maxlength="@maxLength" }
-        @if(largeWidth) { class="textarea--3-4" }
-        >@field.value</textarea>
-    @if(charCount) {
-        <p class="char-counter-text flush js-hidden">@Messages("lbl.1200.character.limit")</p>
-    }
-</div>
+<div class="govuk-character-count" data-module="govuk-character-count" data-maxlength="@charCount">
+    <div class="govuk-form-group">
+        @label
+        @if(hintText.nonEmpty) {
+            <span class="form-hint" id="@{field.id}-hint">@Messages(hintText)</span>
+        }
+        @if(field.hasErrors) {
+            <span class="error-notification" id="@{field.id}-error-notification" data-journey="error - field:user input:@field.name">
+                <span class="visuallyhidden">Error: </span>@field.error
+            </span>
+        }
+        <textarea class="govuk-textarea govuk-js-character-count" id="with-hint" name="@field.name" id="@field.id"
+            @if(hintText.nonEmpty || field.hasErrors) { aria-describedby="@if(hintText.nonEmpty) {@{field.id}-hint} @if(field.hasErrors) {@{field.id}-error-notification}" }
+            @if(cols.nonEmpty) { cols="@cols" }
+            @if(rows.nonEmpty) { rows="@rows" }
+            @if(maxLength.nonEmpty) { maxlength="@maxLength" }
```

Comment:
this maxLength  attribute... do we need it?

Reply:
Removed as not needed